### PR TITLE
implement "check" (make check) into build_desura.sh

### PR DESCRIPTION
**Pull request by matthiaskrgr**

### CODE REVIEW

File: build_desura.sh [hunk 1]
```diff
@@ -21,5 +27,12 @@ if [ ! -d "build" ] ; then
...
 cd build
 cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX -DBINDIR=$BINDIR -DRUNTIME_LIBDIR=$LIBDIR -DDATADIR=$DATADIR -DBUILD_CEF=OFF || exit 3
-make install $@ || exit 4
+make install $args || exit 4
+
+if [[ "$check" == "true" ]] ; then
+	printf "Running 'make check'\n"
+	# already in /build
+	make test
+	./src/tests/UtilLinux_test
```
Comment (karolherbst) [inline build_desura.sh]: Is there any reason why you are calling this test explicitly?

**Comment by karolherbst** [comment]
This patch is breaking if you passing more options to this script:

```
$ ./build_desura.sh -j8 check
Make sure to run sudo ./install-deps.sh before compiling!

We are compiling CEF first
-- using clang
-- detected 64bit
-- Configuring done
-- Generating done
-- Build files have been written to: /home/karol/Dokumente/repos/Desurium/build_cef
make: *** No rule to make target `check'.  Stop.
make: *** Waiting for unfinished jobs....
```

File: build_desura.sh (hunk 1 above)
Comment (matthiaskrgr) [inline build_desura.sh]: If it is included in   make test  I can skip it.
I had in mind that not all tests were included in make test.

**Comment by matthiaskrgr** [comment]
Mmh, I tried to avoid this :/

**Comment by karolherbst** [comment]
tests are declared here: https://github.com/lodle/Desurium/blob/master/src/tests/CMakeLists.txt
so if one of the test wouldn't be included, it wouldn't be build anyway

**Comment by karolherbst** [comment]
but now it works fine

**Pull request merged**